File: app/src/test/java/com/example/mike/mp3player/service/library/utils/MediaLibraryUtilsTest.java
package com.example.mike.mp3player.service.library.utils;

import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import java.io.File;

import static org.junit.Assert.assertEquals;

public class MediaLibraryUtilsTest {

    private final String PARENT_DIR = File.separator + "tmp";
    private final String FILE = "test";
    private final String EXPECTED_APPENDED_PATH = PARENT_DIR + File.separator + FILE;

    @BeforeEach
    public void setUp() {
    }

    @AfterEach
    public void tearDown() {
    }

    @Test
    public void testAppendToPath() {
        assertEquals(EXPECTED_APPENDED_PATH, MediaLibraryUtils.appendToPath(PARENT_DIR, FILE) );
    }

    @Test
    public void testAppendToFilePath() {
        File originalFile = new File(PARENT_DIR);
        File appendedFile = MediaLibraryUtils.appendToFilePath(originalFile, FILE);

        assertEquals(EXPECTED_APPENDED_PATH, appendedFile.getPath());
    }
}

File: service/src/test/java/com/github/goldy1992/mp3player/service/library/utils/MediaLibraryUtilsTest.kt
package com.github.goldy1992.mp3player.service.library.utils

import org.junit.Assert
import org.junit.jupiter.api.AfterEach
import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test
import java.io.File

class MediaLibraryUtilsTest {
    private val PARENT_DIR = File.separator + "tmp"
    private val FILE = "test"
    private val EXPECTED_APPENDED_PATH = PARENT_DIR + File.separator + FILE
    @BeforeEach
    fun setUp() {
    }

    @AfterEach
    fun tearDown() {
    }

    @Test
    fun testAppendToPath() {
        Assert.assertEquals(EXPECTED_APPENDED_PATH, MediaLibraryUtils.appendToPath(PARENT_DIR, FILE))
    }

    @Test
    fun testAppendToFilePath() {
        val originalFile = File(PARENT_DIR)
        val appendedFile = MediaLibraryUtils.appendToFilePath(originalFile, FILE)
        Assert.assertEquals(EXPECTED_APPENDED_PATH, appendedFile.path)
    }
}